### DISCUSSION TITLE
Conda version vs build

### DISCUSSION THREAD

**Discussion by s-canchi** [nih-cfde/training-and-engagement#361] [Conda Q&A] [answered]
What is the difference between version versus build number seen on conda packages ?

**Comment by s-canchi** [comment, marked as answer]
- Version of the software reflects the main updates of the software while builds can imply additional information like build for a particular OS (linux vs Mac) or some patch fixes to a major version or if it built for different versions of software (for e.g. py 2 vs py3)
- More information about what goes into building software for conda which highlights the different aspects like version, build etc: https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html

**Comment by ctb** [comment]
On Thu, Mar 25, 2021 at 02:05:22PM -0700, Saranya Canchi wrote:
> - Version of the software reflects the main updates of the software while builds can imply additional information like build for a particular OS (linux vs Mac) or some patch fixes to a major version or if it built for different versions of software (for e.g. py 2 vs py3)
> - More information about what goes into building software for conda which highlights the different aspects like version, build etc: https://docs.conda.io/projects/conda-build/en/latest/resources/package-spec.html

also note: version is the author's version of the software.

'build' is conda-version specific. For example, sourmash 4.0 has a build 0
and a build 1 from where luiz removed a now-unnecessary installation
requirement.
